actions-cleanup-openstack-vms/readme.md

```diff
@@ -0,0 +1,28 @@
+# Overview
+
+This action is clean up VMs that are older than a certain period of time in openstack
```

Comment:
```suggestion
This action is made to clean up VMs that are older than a certain period of time in an OpenStack cluster.
```

Reply:
And if we say older, it does not mean they are running. So we want to check the creation date.

Reply:
exactly